### DISCUSSION TITLE
Offer Docker Images without having to build them everytime.

### DISCUSSION THREAD

**Discussion by the-hotmann** [ente-io/ente#673] [Enhancements]
Hi,

first: thanks for this, I just read the news and am very hyped. I have a lot of docker images/containers running in my stacks and would also still like to continue to pull them from dockerhub.

AFAIK there currently is not way of doing so, but just a way to pull from GitHub and then build it yourself. In general it would be way more convinient if they would have been build for all and pushed to Docker Hub.

Like this, just a single docker-compose would have been required, nothing more.
Also a direct link to Docker Hub would be awesome.

Thanks in advance!

**Comment by the-hotmann** [comment]
I also want to add, that if pre-build images are offered, maybe also think about:

1. offering an all-in-one image
2. offering a seperate image per service

Like this people can decide if they want to just use one image (for most people more easy and totally ok) or if they want to use a seperate image for each service, to maybe vertically scale it in k8s/kubernetes.

**Comment by monarc83** [comment]
I'm pretty sure this is something a lot of people are looking for, so +1 from me. Are there any plans for this yet?

**Reply by mnvr** [reply]
Plans, yes, but no ETA. We're trying to find out the common denominator - some folks want splits, some want a consolidated image with everything, and some just want the server. Right now, we're leaning towards publishing a standalone image for our server ("museum").

**Reply by the-hotmann** [reply]
@mnvr thanks, thats all I wanted to hear :)
A standalone is pretty good for the start.

Feel free to ping me, once it is released.

**Comment by mnvr** [comment]
This is done. The images are accessible at **ghcr.io/ente-io/server**, e.g.,

```sh
docker pull ghcr.io/ente-io/server
```

There is also a guide on how to use these pre-built images to run the starter docker compose cluster - see [server/docs/docker.md](https://github.com/ente-io/ente/blob/main/server/docs/docker.md).

**Reply by the-hotmann** [reply]
Thanks, I appreciate it :)
Will test it soon and give some feedback

**Reply by spupuz** [reply]
I will test it too!!
